.kokoro/tests/run_single_test.sh

```diff
@@ -41,6 +41,17 @@ fi
 nox -s "$RUN_TESTS_SESSION"
 EXIT=$?
 
+# Inject region tag data into the test log
+XUNIT_PATH="$PWD/sponge_log.xml"
+XUNIT_TMP_PATH="tmp/drift_tmp.xml"
+
+if [ -f "$XUNIT_PATH" ]; then
+    cp $XUNIT_PATH $XUNIT_TMP_PATH
+    cat "$XUNIT_TMP_PATH" | python3.8 "$PARSER_PATH" inject-snippet-mapping "$PWD" > "$XUNIT_PATH"
+else
+    echo "No XUnit output file found!"
+fi
+
```

Comment:
The same question.

If it is reasonable to run this only on a single Python version periodic builds, can we introduce an environment variable for the switch?

Then can we only fetch the parser in `run_tests.sh`?